recipes/asphalt-sqlalchemy/meta.yaml

```
@@ -0,0 +1,39 @@
+{% set version = "5.0.1" %}
+
+package:
+  name: asphalt-sqlalchemy
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/a/asphalt-sqlalchemy/asphalt-sqlalchemy-{{ version }}.tar.gz
+  sha256: fba70f703435b518e20a57bd523203e59aab6e2a4dc3c7d018ab418e49a5a44a
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+
+requirements:
+  host:
+    - python
```

Should be the same as run.

```suggestion
    - python >=3.7
```

---

Thanks!